format/private/format.sh

```diff
@@ -154,15 +117,13 @@ function time-run {
 
 function run-format {
   local lang="$1" && shift
-  local fmtname="$1" && shift
   local bin="$1" && shift
   local args="$1" && shift
   local tuser
   local tsys
 
   local files=$(ls-files "$lang" $@)
   if [ -n "$files" ] && [ -n "$bin" ]; then
-    echo "Formatting ${lang} with ${fmtname}..."
```

Comment:
Was this replaced with something?

Reply:
Yes, multi run prints the label of the target it's about to run

Reply:
Advantage is that when we buffer stdout of this tool we don't lose the print statement at the start time